src/components/apps/details/Drawer.js

```diff
@@ -98,33 +99,31 @@ function DetailsHeader({
     const [open, setOpen] = useState(false);
 
     useEffect(() => {
-        const protocol = window.location.protocol.concat("//");
-        const host = protocol.concat(window.location.host);
-        setLink(
-            host.concat(`/${NavigationConstants.APPS}/${systemId}/${appId}`)
-        );
+        const host = getHost();
+        setLink(`${host}/${NavigationConstants.APPS}/${systemId}/${appId}`);
```

Comment:
Since this formatted string building is repeated 3 times in this PR, it should probably be refactored into a util function.